lib/samples/show_device_location_history/README.md

```diff
@@ -0,0 +1,43 @@
+# Show device location history
+
+Display your device's location history on the map.
+
+![Image of show device location history](show_device_location_history.png)
+
+## Use case
+
+You can track device location history and display it as lines and points on the map. The history can be used to visualize how the user moved through the world, to retrace their steps, or to create new feature geometry. An unmapped trail, for example, could be added to the map using this technique.
+
+## How to use the sample
+
+Tap 'Start tracking' to start tracking your location, which will appear as points on the map. A line will connect the points for easier visualization. Tap 'Stop tracking' to stop updating the location history. This sample uses a simulated data source to allow the sample to be useful on desktop/non-mobile devices. To track a user's real position, use the `LocationDataSource` instead.
```

Comment:
The "T" in "Tracking" should be capitalized in 'Start tracking' and 'Stop tracking'